src/Redux/Store/TurnOrder/Configuration/index.ts

```diff
@@ -4,7 +4,7 @@ import { createSelector } from 'reselect'
 import { get as getFromDb, set as setToDb } from 'idb-keyval'
 
 import config from '../../../../config'
-import { ITurnOrderSetup, Mode } from '../../../../types'
+import { ITurnOrderPlayerCount, ITurnOrderSetup, Mode } from '../../../../types'
 import { RootState } from '../../'
 import {
   actions as activeGameActions,
```

Comment:
change the `TURNORDER_CONFIG_DB_KEY` so users do not get crashes because of different settings inside indexedDB